.travis.yml

```diff
@@ -80,7 +80,7 @@ install:
 
 script:
     - ant -q clean
-    - travis_wait 120 "./.travis/antTest.sh"
+    - "./.travis/antTest.sh"
```

Comment:
https://app.travis-ci.com/github/apache/tomcat/jobs/531899762 (the build for this PR) fails with `No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.`. So yes, `travis_wait` is needed!
Maybe the duration should be reduced though.
AFAIK Travis jobs could run for at most 50 mins but I see some which succeeded even after that...

Reply:
This command will not exceed 60 mins? [https://app.travis-ci.com/github/apache/tomcat/jobs/529778525](https://app.travis-ci.com/github/apache/tomcat/jobs/529778525)

Reply:
I cannot say how long it might take. It depends on how loaded the Travis VM is. Sometimes jobs take twenty something minutes, sometimes fifty something minutes...
According to https://docs.travis-ci.com/user/customizing-the-build/#sts=Build%20Timeouts the max timeout for a job should be 50 mins, but it seems `travis_wait` may extend it. I cannot find any reference in their docs for its maximum.